### ISSUE TITLE
Custom range selection no longer works

### ISSUE THREAD

**Issue by morrow95** [dangrossman/bootstrap-daterangepicker#744]
I was previously using 1.3.21 and just loaded 2.0.5.  The custom range selection no longer works since the update.  I came back and looked at your demo/live examples and it does not work in them either.  All other selections (all, today, yesterday, etc) work fine still.

Ideas?

**Comment by dangrossman** [comment]
Could you explain "doesn't work"? I just clicked on custom date at the daterangepicker.com example and the two calendars appeared as expected.

**Comment by morrow95** [comment]
Looking at it further it does 'work', but is very confusing display-wise as well as how it operates now.

Why can you set both the start and end 'in' the start frame?  You can do the same in the end frame, but it displays them in the start frame.

Aside from that, what I expect is when I am hovering over the start frame it only changes the start frame.... when I am hovering over the end frame it only changes the end frame.

I guess what I expect is when I click/hover over the start frame it only changes it - same goes for the end frame.  It is natural to look at the viewable dates of each frame to know what range you are selecting rather than look at the actual displayed months and highlighted days.  It just seems very unintuitive compared to how it used to be as none of the above issues were present (at least with 1.3.21).

It looks as if you changed the backend scripting to keep track of clicks to set a date rather than what actual frame they occur in.  It is very easy to be hovering over the start frame and it changes the display in the end frame and vice versa if you click around.

**Comment by dangrossman** [comment]
I changed it because this is how every other date range picker on the web works, and the inconsistency was routinely brought up. Every travel site, hotel reservations, appointment setting, calendar application I tried worked this way: the first click selects the beginning of a range, and the second the end of the range, regardless of which calendar you start with.

**Comment by morrow95** [comment]
Fair enough... I guess it just seems the complete opposite to me... maybe it is because I use time as well.  So even if you use the same calendar to set your dates you still set the times on their respective 'sides'.

Off the top of my head I looked at Orbitz and Travelocity as I have used those both before.  While they do work similar to how you describe one of the main differences is they aren't showing a start and end calendar, but just show a two month view... paging for months changes both as a whole not each separately.  Travelocity takes it a step further and has you click on a start and end input field to fill in separately.

Anyways, sorry for the confusion... it just struck me as off I guess as I was used to the other format.  Just different ways of looking at it I suppose.
